### PR TITLE
Validator: add the Persian translations

### PR DESCRIPTION
**Pull request by behnoushnorouzi**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no 
| Deprecations? | no 
| Tests pass?   | yes 
| Fixed tickets | #30162
| License       | MIT
| Doc PR        | -

Add the missing translations for the Persian (fa) validator.

### CODE REVIEW

**Comment by behnoushnorouzi** [comment]
status:Needs Review

**Review by javiereguiluz — APPROVED** [review]
Thanks @behnoushnorouzi for your contribution!

**Comment by fabpot** [comment]
Thank you @behnoushnorouzi.

File: src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf [hunk 1]
```diff
@@ -278,6 +278,58 @@
...
+            <trans-unit id="82">
+                <source>Error</source>
+                <target>خطا</target>
+            </trans-unit>
+            <trans-unit id="83">
+                <source>This is not a valid UUID.</source>
+                <target>این مقدار یک UUID درست نیست.</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
```
Comment (javiereguiluz) [inline src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf]: Just asking: in the other translations you maintained the dot (`.`) of the original translation. Should we keep it also for id=84 and id=85? Thanks.

Comment (behnoushnorouzi) [reply src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf]: yes, it's better we keep it but sorry I forgot it

Comment (javiereguiluz) [reply src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf]: This pull request is now merged, so we cannot change it. But don't worry, our mergers (@fabpot for example) can fix this directly in the repository. Thanks for your contribution.

Comment (fabpot) [reply src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf]: Done in 6c0190eee8d565bdd8bc33515e9ee67169529d56

Comment (behnoushnorouzi) [reply src/Symfony/Component/Validator/Resources/translations/validators.fa.xlf]: @fabian thank you

**Pull request merged**